portfolio/src/main/webapp/comment.js

```diff
@@ -23,15 +23,12 @@ function getLoginStatus() {
 /**
  * Shows the UI for logged in member.
  * @param {string} user User name.
- * @param {string} loggingUrl Log in or log out url.
+ * @param {string} loggingUrl Log out url.
  * @param {html element} commentSection HTML div element for comment section.
  */
 function showMemberUI(user, loggingUrl, commentSection) {
-    const logoutDiv = document.createElement("div");
-    let id = document.createAttribute("id");
-    id.value = "logout";
-    logoutDiv.setAttributeNode(id);
-    logoutDiv.innerHTML = getGreetingHTML(user, loggingUrl);
+    let htmlText = getGreetingHTML(true, user, loggingUrl);
```

Comment:
Nit: If you want to pass boolean here: /* isLoggedIn= */ true

Reply:
Fixed